File: HactoberFest/Python/linear_search.py
def linear_search(arr, target):
    for index, item in enumerate(arr):
        if item == target:
            return index  # Return the index of the target element if found
    return -1  # Return -1 if the target element is not in the list

# Example usage
arr = [3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5]
target = 9

result = linear_search(arr, target)

if result != -1:
    print(f"Element {target} found at index {result}.")
else:
    print(f"Element {target} not found in the list.")

File: Java/linearSearch.java
public class Main{
    public static void main(String[] args){
        int[] nums={23,45,1,2,8,19,-3,16,-11,28};
        int target=19;
        int ans=linearSearch(nums,target);
        System.out.println(ans);
    }
  static int linearSearch(int[] arr,int target) {
        if (arr.length == 0) {
            return -1;
        }
        for(int index=0;index<arr.length;index++){
            //check for element at every index if it is = target
            int element=arr[index];
            if(element == target){
                return index;
            }
        }
        //this line will execute if none of the return statements have executed
        //hence the target not found
        return -1;
    }
}
